src/test/java/org/stellar/sdk/TransactionTest.java

```diff
@@ -108,6 +108,20 @@ public void testBuilderSuccessPublic() throws FormatException {
             transaction.toEnvelopeXdrBase64());
   }
 
+  @Test
+  public void testTransactionFromEnvelopeXdrBase64() {
+    String xdrBase64Envelope;
+    Transaction tx;
+
+    xdrBase64Envelope = "AAAAAF7FIiDToW1fOYUFBC0dmyufJbFTOa2GQESGz+S2h5ViAAAAZAAKVaMAAAABAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAA7eBSYbzcL5UKo7oXO24y1ckX+XuCtkDsyNHOp1n1bxAAAAAEqBfIAAAAAAAAAAABtoeVYgAAAEDzfR5PgRFim5Wdvq9ImdZNWGBxBWwYkQPa9l5iiBdtPLzAZv6qj+iOfSrqinsoF0XrLkwdIcZQVtp3VRHhRoUE";
```

Comment:
Would you mind constructing the transaction here instead of using XDR? It's easier to check quickly if it tests ex. time bounds.

Reply:
Will do.

Reply:
@bartekn: I've made the changes your requested and need advice on how to proceed. 

I would like to update the files in this PR, but am not sure if that is possible now. The PR predated the `android` branch. Now I am working on the `android` branch (I have pulled this branch and merged my code into a local copy of the `android` branch), so the last changes are based upon the `android` branch. I guess I need to wait until `android` is merged into `master`, right?

Reply:
Can you update your master branch (the branch used in this PR) with the changes? I don't think working on a PR in 2 branches is a good idea because then we'll have to wait for one branch to be merged before another.